packages/token/src/tests/erc1155.cairo

```diff
@@ -1,6 +1,10 @@
-pub(crate) mod common;
+pub mod common;
```

Comment:
It might be worth rethinking if we should have a public `common/` specifically for tests given the new structure. Not a pressing issue ATM though

Reply:
I agree, let's just not delay merging this PR because of it, and we can revisit it in the future.